Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
I'd rather we didn't use a build that requires doing the whole build process twice. Let users pick static or shared libraries rather than hacking a way to install both.